trajectory_generator:
  ros__parameters:
    waypoints_file: "waypoints.yaml"
    bspline_degree: 3
    bspline_samples: 300
    control_point_spacing: 0.15
    control_point_multiplier: 4
    max_velocity: 0.3
    min_velocity: 0.05
    max_acceleration: 0.08
    max_jerk: 0.15
    sampling_interval: 0.03 